nitorch/nn/training/_training.py

```diff
@@ -582,7 +598,7 @@ def init(self):
             self.save_random_state()
             self.epoch = self.initial_epoch
             self.model.to(dtype=self.dtype, device=self.device)
-            self._eval(self.epoch)
+            _ = self._eval(self.epoch)
```

Comment:
Do we need that? I thought we could call functions without return values even if the do return one

Reply:
Removed.